packages/dev/test/NetworkSimultaion.test.ts

```diff
@@ -161,7 +154,48 @@ contract('Network Simulation for Relay Server', function (accounts) {
       })
     })
 
-    async function sendMultipleRelayedTransactions (): Promise<void> {
+    describe('boosting & rebroadcasting all transactions', function () {
+      let id: string
+      before(async () => {
+        id = (await snapshot()).result
+      })
+      after(async () => {
+        await revert(id)
+      })
+      it('should boost underpriced transactions and only rebroadcast fairly priced transactions', assertBoostAndRebroadcast)
+      it('should not boost any transactions if latest on-chain nonce is lower than oldest pending tx nonce', async function () {
```

Comment:
Description is unclear. I assume it says something more in line with "should throw exception if there is a gap between last saved and last mined transaction that we don't know hot to fill, so there is no way to re-use this worker ever again", right?

Reply:
Yes, but I didn't want to right a book in the description

Reply:
At least write "should throw an exception", please :-)